db/migrations/20201109145014_add_feedbacks_table.js

```diff
@@ -0,0 +1,22 @@
+const { addUpdateTsTrigger } = require("../util")
+
+exports.up = async (knex) => {
+  await knex.schema.createTable("feedbacks", (table) => {
+    table.increments()
+    table
+      .integer("user_id")
+      .notNullable()
+      .references("id")
+      .inTable("users")
+      .onDelete("CASCADE")
+      .index()
+    table.string("status").notNullable()
```

Comment:
We should include a reference to nonnullable reference to `website_id`, whose domain should be included in the incoming request. The SQL I slacked you creates the website if it doesn't already exist.